src/test/java/com/googleinterns/zoomtube/servlets/DiscussionServletTest.java

```diff
@@ -118,8 +175,7 @@ public void doPost_reply_storesCommentWithAllProperties() throws ServletExceptio
   public void doPost_rootComment_storesTypeAndTimestamp() throws ServletException, IOException {
     testServices.setEnvIsLoggedIn(true);
     when(request.getParameter(DiscussionServlet.PARAM_LECTURE)).thenReturn(LECTURE_ID_STR);
-    when(request.getParameter(DiscussionServlet.PARAM_TIMESTAMP))
-        .thenReturn(TIMESTAMP_MS_STRING);
+    when(request.getParameter(DiscussionServlet.PARAM_TIMESTAMP)).thenReturn(TIMESTAMP_MS_STRING);
```

Comment:
TIMESTAMP_MS_STRING and LECTURE_ID_STR add complexity. It is much easier to read the following:

when(request.getParameter(DiscussionServlet.PARAM_LECTURE)).thenReturn("1");
    when(request.getParameter(DiscussionServlet.PARAM_TIMESTAMP))	    when(request.getParameter(DiscussionServlet.PARAM_TIMESTAMP)).thenReturn("2000");

assertThat(comment.timestampMs().get()).isEqualTo(2000);

It is easy to see that "2000" and 2000 are the same. TIMESTAMP_MS_STRING and TIMESTAMP_MS are 2 different variables and therefore I need to look at their declaration to understand how they are related.

Reply:
Right.  Removed!